src/clib/pioc_support.c

```diff
@@ -2759,6 +2757,12 @@ int PIOc_createfile_int(int iosysid, int *ncidp, int *iotype, const char *filena
          */
         comm = ios->union_comm;
     }
+
+    /* Initialize the multi-variable cache in the file that is used to cache
+     * data from multiple variables before writing it out to the output file
+     */
+    spio_file_mvcache_init(file);
```

Comment:
ok, I will remove init'ing mvcache for ADIOS

Reply:
Since there are many instances (based on iotype, file open modes etc) where the MVCache is not used as we discussed offline it might make sense to init it all the time (like in the current version of the code in the branch) and add some more comments on when its used

Reply:
So leaving the code unchanged and adding some more comments